data/json/recipes/armor/head.json

```diff
@@ -92,6 +92,19 @@
     "components": [ [ [ "felt_patch", 8 ] ] ]
   },
   {
+    "result": "crown_golden_survivor",
+    "type": "recipe",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_HEAD",
+    "skill_used": "fabrication",
+    "difficulty": 6,
+    "time": 600000,
+    "book_learn": [ [ "jewelry_book", 6 ] ],
+    "qualities": [ { "id": "HAMMER", "level": 3 } ],
+    "tools": [ [ [ "surface_heat", 10, "LIST" ] ] ],
```

Comment:
`surface_heat` group include `charcoal smoker` and `fire nearby` tools. They don't fit as metalworking tools here. Better explicitly  replace them by `soldering iron` and `integrated toolset`.
The same for ring crafting below.